src/actions/task.js

```diff
@@ -29,13 +29,13 @@ export const fetchTeamMembersTask = (currentUserId, authenticatedUserId) => asyn
   try {
     const state = getState();
     //The userId will be equal the currentUserId if provided, if not, it'll call the selectFetchTeamMembersTaskData, that will return the current user id that's on the store
-    
+
     const userId = currentUserId ? currentUserId : selectFetchTeamMembersTaskData(state);
     const authUserId = authenticatedUserId ? authenticatedUserId : null
     console.log(authUserId)
-    
+
```

Comment:
```suggestion
```

Reply:
Please explain your suggestion for this line. The change for this line was empty spaces being removed automatically by the formatter. I'm not sure whether your suggestion means adding back the empty spaces.